dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - sample_id
+          - activity_segment
+          - normalized_app_name
+          - normalized_channel
+          - country
+          - city
+          - days_seen_bits
+          - days_since_first_seen
+          - days_since_seen
+          - device_model
+          - is_new_profile
+          - first_seen_date
+          - locale
+          - normalized_os
+          - durations
+          - normalized_os_version
+          - app_version
+          - os_version_major
+          - os_version_minor
+          - os_version_patch
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_channel
+        expected_values:
```

Comment:
Would the upper/lower case be relevant?
I'd say if we took the set of unique values and the channel is automatically generated by the app/ping then it shouldn't.

Reply:
I think it does matter, if suddenly app changed the casing we should know about it as downstream tables might be doing direct string comparison and then it'd matter.